cdat/tests/test_DictCombo.py

```diff
@@ -89,14 +89,14 @@ def test_duplicates(editors):
         # test duplicate items
 
         if index == 0:
-            d_e.key_value_rows[0].setKey(key_list[5])
+            d_e.key_value_rows[0].setKey(key_list[4])
         else:
-            d_e.key_value_rows[1].setKey(key_list[1])
+            d_e.key_value_rows[1].setKey(key_list[2])
 
         if index == 0:
-            assert d_e.key_value_rows[0].key() == "quesadilla"
+            assert d_e.key_value_rows[0].key() == "taco"
```

Comment:
haha yep it was because I now have the dictionary display values sorted by the key so it restructured how my tests were testing for duplicate values